### ISSUE TITLE
TypeError: _reactDom2.default.render

### ISSUE THREAD

**Issue by taqimustafa** [kentcdodds/react-testing-library#191]
- `react-testing-library` version: 5.1.1
- `react` version: 16.2.0
- `node` version: 8.11.4
- `npm` version: 5.6.0

### Relevant code or config:

```js
import React from 'react';
import 'jest-dom/extend-expect';
import { render } from 'react-testing-library';

describe('Button Component', () => {
  it('renders correctly', () => {
    render(<div>React Testing Library</div>);
  });
});

```

### What you did:
Trying to add test for components added in storybook
https://github.com/storybooks/storybook

### What happened:
Getting this error
`TypeError: _reactDom2.default.render is not a function`
<img width="604" alt="screenshot 2018-10-04 at 12 36 55 pm" src="https://user-images.githubusercontent.com/8524942/46459396-40431d80-c7d2-11e8-9c3f-597abd6d2ff5.png">

### Reproduction:
Not sure if i can reproduce storybook in codesandbox 😞 

### Suggested solution:
Not sure, but i guess issue is with the configuration for storybook ? Tried for another repo that doesn't use storybook, and it works like a charm.

**Comment by gnapse** [comment]
I don't understand what storybook stories for a component have to do with testing that component. These are two independent things. You have the component implementation, which you import in your test file, for testing. This is where you use react-testing-library, and this is what your code examples above show. And then you have a storybook stories file, where you also import your component, to render it and showcase what it does. But on this side react-testing-library has nothing to do with it, does it?

**Comment by kentcdodds** [comment]
Yes, this is pretty strange... Could you do some more digging please?

**Comment by taqimustafa** [comment]
@gnapse Makes sense 💯, i was using storybook for component implementation & also for show casing, i guess i need to separate those to reduce complexity. Thanks for the suggestion. 

But there are few addons that storybook provides for testing.
https://github.com/storybooks/storybook/tree/master/addons/jest
I was using those for testing component with enzyme.

**Comment by kentcdodds** [comment]
Hmmm... I feel like if you can make that plugin work with enzyme it should be even easier to make it work with react-testing-library. I'm not sure what's going on, but if you'd like to dig in a little bit it'd be a great addition to the docs.

**Comment by taqimustafa** [comment]
Will look into that 👍and will update you if there is any progress.

**Comment by bhagatali** [comment]
@taqimustafa did you ever find what the cause for this is? I am seeing a similar problem and could use some pointers.

**Comment by bhagatali** [comment]
Never mind, I found the problem. This was caused because there was a `jest` mock of the `react-dom` to stub out the `findDOMNode` that was causing this problem. We revisited the way the mock was being performed and it works now.
